tests/e2e/configurer/chain/commands.go

```diff
@@ -689,7 +689,7 @@ func (n *NodeConfig) SendIBC(dstChain *Config, recipient string, token sdk.Coin)
 				return false
 			}
 		},
-		time.Minute,
+		3*time.Minute,
```

Comment:
Note to reviewer: This is a drive by "fix". It became apparent to me that some packets get stuck, so a minute at times in not enough to wait for the result on the receiving chain.